service/src/main/java/gov/nasa/pds/api/registry/model/Pds4ProductBusinessObject.java

```diff
@@ -93,11 +93,8 @@ public int setResponse(HitIterator hits, Summary summary, List<String> fields, b
         {
             uniqueProperties.addAll(ProductBusinessObject.getFilteredProperties(kvp, fields, null).keySet());
 
-            if (!onlySummary)
-            {
-            	Pds4Product prod = Pds4ProductFactory.createProduct(hits.getCurrentId(), kvp, this.isJSON);
-            	list.add(prod);
-            }
+			Pds4Product prod = Pds4ProductFactory.createProduct(hits.getCurrentId(), kvp, this.isJSON);
```

Comment:
why is the identation so far on the right ?

Reply:
Probably something I missed, and I haven't got my autoformatter set up yet - will fix

Reply:
Actually this wasn't a new change - there's some mixing of spaces and tabs throughout this file.  I've reformatted that function, but can expand the scope if desired.